.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+go:
+        - 1.3
```

Comment:
Vulcand does not support 1.3 actually, I suggest sticking with 1.4 for now, can you please make the change?